### PR TITLE
Bug 1910140: fix the api dashboard with changes in upstream kube 1.20

### PR DESCRIPTION
**Pull request by tkashem**
- adjust graph titles 
- update the dashboard with changes in kube 1.20
- add new metrics to the dashboard

### CODE REVIEW

File: manifests/0000_90_kube-apiserver-operator_05_api_performance_dashboard.yaml [hunk 1]
```diff
@@ -209,7 +209,7 @@ data:
           "timeFrom": null,
           "timeRegions": [],
           "timeShift": null,
-          "title": "etcd Request Duration - 99th Quantile",
+          "title": "etcd Request Duration - 99th Percentile",
```
Comment (sttts) [inline manifests/0000_90_kube-apiserver-operator_05_api_performance_dashboard.yaml:212]: part of first commit?

**Comment by sttts** [comment]
Second commit seems to have many unrelated changes

File: manifests/0000_90_kube-apiserver-operator_05_api_performance_dashboard.yaml (hunk 1 above)
Comment (tkashem) [reply manifests/0000_90_kube-apiserver-operator_05_api_performance_dashboard.yaml:212]: @sttts this is part of the second commit - it's either 0.99 quantile or 99th percentile.

**Comment by tkashem** [comment]
@sttts in the second commit I did the following:
- upstream p&f changed the label values for the metrics, I had to fix it.
- there are a number of new metrics, I added these to the dashboard
- I added the metrics for `watch`.

**Pull request merged**